controllers/news-controller.js

```diff
@@ -53,4 +54,21 @@ const { selectTopics } = require('../models/topics-model');
       res.status(200).send({ comments });
     })
     .catch(next);
-  };
\ No newline at end of file
+  };
+
+  exports.postComment = (req, res, next) => {
+    const {article_id} = req.params
+    const {username, body} = req.body
+    selectArticleById(article_id)
+    .then(() => {
+        return selectUserByUsername(username)
+    })
+    .then(() => {
+        return insertComment({article_id, username, body})
+    })
+    .then((comment) => {
+        res.status(201).send({ comment });
+    })
```

Comment:
This this does work, but you dont actually need to have separate checks for if the article and user are valid. By only having the insertComment query here psql will throw an errror for you if those two values are not valid